codegen/generator/nodejs/templates/src/method_comment.ts.tmpl

```diff
@@ -4,26 +4,56 @@
 	{{- $argsDesc := ArgsHaveDescription .Args }}
 	{{- $deprecationLines := FormatDeprecation .DeprecationReason }}
 	{{- if or .Description $argsDesc .IsDeprecated }}
-		{{- $commentLines := CommentToLines .Description }}
+		{{- $desc := CommentToLines .Description | GetParamDescription }}
 		{{- /* we split the comment string into a string slice of one line per element */ -}}
 {{""}}
 {{""}}
   /**
-		{{- range $commentLines }}
+		{{- range $desc }}
    * {{ . }}
 		{{- end }}
 	{{- end }}
 
 	{{- range $required }}
 		{{- if .Description }}
-   * @param {{ .Name }} {{ .Description }}
+		{{- /* Reference current arg to access it in range */ -}}
+		{{- $arg := . }}
+		{{- /* Write argument description */ -}}
+		{{- $desc := CommentToLines .Description | GetParamDescription }}
+		    {{- range $i, $line := $desc }}
+		        {{- /* If it's the first line, add the JSDoc tag, otherwise treat it as a simple line */ -}}
+		        {{- if (eq $i 0) }}
+   * @param {{ $arg.Name }} {{ $line }}
+		        {{- else }}
+   * {{ $line }}
+		        {{- end }}
+		    {{- end }}
+
+		    {{- /* Write argument examples */ -}}
+		    {{- $examples := CommentToLines .Description | GetParamExample }}
+		    {{- range $examples }}
+   * @example {{ . }}
+		    {{- end }}
 		{{- end }}
 	{{- end }}
 
 	{{- if ArgsHaveDescription $optionals }}
 		{{- range $optionals }}
 			{{- if .Description }}
-   * @param opts.{{ .Name }} {{ .Description }} 
+		        {{- /* Reference current arg to access it in range */ -}}
```

Comment:
indent 1 level under the `if` block

Reply:
Good catch! I'm waiting for feedback on https://github.com/dagger/dagger/pull/4367#issuecomment-1382619290 before changing this.

Reply:
I think this is not impacted by the linked comment and should be indented 1 level under the `if`